lunes_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -35,6 +35,10 @@ msgstr "Vokabeln"
 msgid "api keys"
 msgstr "API Keys"
 
+#: cms/admin.py:36
+msgid "feedback"
+msgstr ""
```

Comment:
```suggestion
msgstr "Feedback"
```
Even though we don't translate this string, our translation checking tool expects something here.